test/integration/features/basic.feature

```diff
@@ -147,6 +147,12 @@ Feature: Basic
      """
      exposed
      """
+
+  Scenario: User can access ruby-ex route
+   Given Minishift has state "Running"
+    When status code of HTTP request to access "ruby-ex" application url in namespace "ruby" is equal to "200"
+    Then body of HTTP request to access "ruby-ex" application url in namespace "ruby" contains "Welcome to your Ruby application on OpenShift"
+     And body of HTTP request to access "ruby-ex" application url in namespace "ruby" contains "<title>Welcome to OpenShift</title>"
```

Comment:
Here the idea of matching two content is to maximize the scenario accuracy. That does not mean that we should validate all the page content, but the minimum content need to be validated. Which would be enough to identify the HTTP response of the deployed application webpage. IMO these two parameters ```page title``` and some part of ```page body``` would be good enough to increase the response accuracy.